lib/web_ui/lib/src/engine/semantics/scrollable.dart

```diff
@@ -79,6 +99,13 @@ class Scrollable extends RoleManager {
 
   @override
   void update() {
+      // We neutralize the scroll position after all children have been
+      // updated. Otherwise the browser does not yet have the sizes of the
+      // child nodes and resets the scrollTop value back to zero.
+      semanticsObject.owner.addOneTimePostUpdateCallback(() {
```

Comment:
Is `addOneTimePostUpdateCallback` still necessary?

Reply:
Several tests will fail if we call `_neutralizeDomScrollPosition()` directly here without using `addOneTimePostUpdateCallback()`. I think we still need to wait for the update to finish before attempting to neutralize the scroll position.